### DISCUSSION TITLE
vm image not being updated on website

### DISCUSSION THREAD

**Discussion by thibaultmol** [nextcloud/all-in-one#2187] [Questions] [answered]
Just letting you know that the link on this page:
https://nextcloud.com/install/#instructions-server
Which goes to https://download.nextcloud.com/aio-vm/
seems to have an old version from last year instead of the up to date aio vm image

**Comment by szaimen** [comment, marked as answer]
Thanks! However AIO updates itself after you open the aio interface the first time. So it is not really outdated. We'll only update the image once the base-OS is oudated, e.g. when Ubuntu 24.04 LTS is out or if there is some other change that cannot be handled internally.
